provider/vsphere/environ_availzones_test.go

```diff
@@ -53,18 +68,46 @@ func (s *environAvailzonesSuite) TestAvailabilityZones(c *gc.C) {
 	c.Assert(zones[4].Name(), gc.Equals, "z2/child/nested/other")
```

Comment:
You added a zone (len == 6), but you didn't add a test for what zones[5].Name() is.

Reply:
Thanks, I did miss that!